src/main/kotlin/Strategy/strategies/GeneralStrategy.kt

```diff
@@ -1,11 +1,3 @@
 package Strategy.strategies
 
-import Strategy.base.IvaStrategy
-
-class GeneralStrategy : IvaStrategy {
-
-  override fun applyIva(cost: Double): Double {
-    return cost + cost * 0.21
-  }
-
-}
+val GeneralStrategy : (Double) -> Double = { it + it * 0.21 }
```

Comment:
We don't really need to make a file just for this, but it's clearer for comparing with traditional way of writing the pattern.